community/kernel/src/main/java/org/neo4j/kernel/configuration/Config.java

```diff
@@ -565,6 +565,59 @@ public void setLogger( Log log )
     }
 
     /**
+     *
+     * TODO: DRAGONS! This code works for the current dynamic variables(3), but is not generic.
+     * TODO: DRAGONS! No migration or config validation is done.
+     *
+     * @param setting The setting to set to the specified value.
+     * @param value The new value to set, passing {@code null} or empty should reset the value back to default value.
+     * @throws IllegalArgumentException if the provided setting is unknown or not dynamic.
+     * @throws InvalidSettingException if the value is not formatted correctly.
+     */
+    public void setConfigValue( String setting, String value ) throws IllegalArgumentException, InvalidSettingException
```

Comment:
i think we need to rename the method to highlight that it's not a generic set of config value, but a specific update of dynamic properties for runtime only (still works for non-runtime, but targeted use case is runtime update)